x-pack/plugins/alerting/server/usage/alerts_telemetry.ts

```diff
@@ -10,7 +10,7 @@ const alertTypeMetric = {
   scripted_metric: {
     init_script: 'state.types = [:]',
     map_script: `
-      String alertType = doc['alert.alertTypeId'].value;
+      String alertType = doc['alert.alertTypeId'].value.replace(".", "");
       state.types.put(alertType, state.types.containsKey(alertType) ? state.types.get(alertType) + 1 : 1);
```

Comment:
Is there a way of testing this?
Looking at it I though "wait, what is the key in `state.types`? Is it with a "." or not? Won't this break?" and so I looked at the code - and realised this is some kind of injected script.

I have no idea how this works, but I'd feel much more comfortable if this had some kind of a test ensuring the change is safe. :/

Any chance we can add a couple?

Reply:
I will provide a few. Yesterday did the late fix - wasn't sure if it urgent, so pushed as it is.